tests/test_benchmarks/test_imagenet_c.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import pytest
+from pytest import approx
+
+from brainio_base.assemblies import BehavioralAssembly
+from brainscore.benchmarks.imagenet import Imagenet2012
+from brainscore.benchmarks.imagenet_c import Imagenet_C_Individual, Imagenet_C_Noise, Imagenet_C_Blur, Imagenet_C_Weather, Imagenet_C_Digital
+from brainscore.model_interface import BrainModel
+
```

Comment:
the tests in this file fail because the stimulus_sets are currently private and the test is not marked as `pytest.mark.private_access` (see [here](https://github.com/brain-score/brain-score/blob/master/tests/README.md#markers)).
You can either tell me to make the stimulus_sets public (if we're allowed to distribute them publicly) or add the mark